### PR TITLE
Demisto-sdk lint conf change

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

# Description
The changes that should be done before releasing demsito-sdk with new lint command:
1. Build configuration.
2. Pre-commit changes.
3. Dev-requirements should be change.

### CODE REVIEW

**Comment by GalRabin** [comment]
> We have twice: `Run Unit Testing and Lint` in config.yml. Please modify the existing step.
@glicht 
Fixed

File: .circleci/config.yml [hunk 1]
```diff
@@ -215,33 +215,29 @@ jobs:
...
                 exit 0
             fi
-            echo "demisto-sdk version: $(demisto-sdk --version)"
-            echo "mypy version: $(mypy --version)"
-            echo "flake8 py2 version: $(python2 -m flake8 --version)"
-            echo "flake8 py3 version: $(python3 -m flake8 --version)"
-            echo "bandit py2 version: $(python2 -m bandit --version 2>&1)"
-            echo "bandit py3 version: $(python3 -m bandit --version 2>&1)"
-            echo "vulture py2 version: $(python2 -m vulture --version) 2>&1"
-            echo "vulture py3 version: $(python3 -m vulture --version) 2>&1"
```
Comment (glicht) [inline .circleci/config.yml:237]: Why did you delete all these lines? Please bring them back. It is important when debugging to understand the exact version being used of each lint component.

File: .circleci/config.yml [hunk 2]
```diff
@@ -215,33 +215,29 @@ jobs:
...
+            mkdir ./unit-tests
             SHOULD_LINT_ALL=$(./Tests/scripts/should_lint_all.sh)
             if [ -n "$SHOULD_LINT_ALL" ]; then
               echo -e  "----------\nLinting all because:\n${SHOULD_LINT_ALL}\n----------"
-              demisto-sdk lint --run-all-tests --parallel --outfile ./artifacts/failed_unittests.txt -v
+              demisto-sdk lint -p 8 -a -q --test-xml ./unit-tests --log-path ./artifacts --json-report ./artifacts
             else
-              demisto-sdk lint --git --parallel --outfile ./artifacts/failed_unittests.txt -v
+              demisto-sdk lint -g -v --test-xml ./unit-tests --log-path ./artifacts --json-report ./artifacts
```
Comment (glicht) [inline .circleci/config.yml:234]: Shouldn't we run here with `-p`?

Comment (GalRabin) [reply .circleci/config.yml:234]: yes, we can you right, I will change it

File: .circleci/config.yml (hunk 1 above)
Comment (GalRabin) [reply .circleci/config.yml:237]: I added them back.

**Comment by GalRabin** [comment]
Performed another test - to verify that I didn't break anything with this morning minor change - replace Bitwise operator.
